src/components/Event/Event.driver.ts

```diff
@@ -9,6 +9,10 @@ export interface EventDriver extends BaseUniDriver {
   isTimeShown(): Promise<boolean>;
   isSelected(): Promise<boolean>;
   isFullday(): Promise<boolean>;
+  hasOnClick(): Promise<boolean>;
```

Comment:
Instead of `hasOnClick` I would create a method:
```ts
    isButton(): Promise<boolean>;
```

and then I would implement it like this:
```ts
isButton: async () => {
  return (await base._prop('tagName')).toLowerCase() === 'button';
}
```

Reply:
btw, see this: https://github.com/wix-incubator/unidriver/blob/958388bd0de4f648972847451a0574fbd9d127d7/core/src/index.ts#L47